cli/popper/commands/cmd_env.py

```diff
@@ -1,12 +1,14 @@
 import click
+import json
+import urllib2
```

Comment:
We need to add these new dependencies to the `setup.py` file, as well as [vendor them](https://github.com/systemslab/popper/blob/master/cli/bin/popper) (add the corresponding projects as submodules and load their librarly paths to python's syspath).